library/ceph_pool.py

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/python
+#coding: utf-8 -*-
+
+DOCUMENTATION = """
+---
+author: Michael Sambol
+module: ceph_pool
+short_description: Creates ceph pool and ensures correct pg count
+description:
+  There are three possible outcomes:
+    1/ Create a new pool if it doesn't exist
+    2/ Set the pool's pg count to correct number
+    3/ Nothing: pool is created and pg count is correct
+options:
+  pool_name:
+    description:
+      - The pool in question: create it or ensure correct pg count
+    required: true
+  constant:
+    description:
+      - A constant multiplier used in calculating pg count
+    required: true
+  copies:
+    description:
+      - The number of copies the cluster keeps
+    required: true
```

Comment:
You have a point.. Here's how I called this in my testing. I could hardcode the 3 in three in there if you want?

```
- name: create openstack pool
  ceph_pool:
    pool_name: "{{ ceph_common.pool_name}}"
    constant: "{{ ceph_common.pool_constant }}"
    copies: "{{ ceph_common.pool_default_size }}"
  register: pool_output
  run_once: true
  delegate_to: "{{ groups[ceph.monitor_group_name][0] }}"
```